unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Utils/ECSComponentsUtils.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using DCL.Configuration;
+using DCL.Helpers;
+using DCL.Models;
+using UnityEngine;
+
+public static class ECSComponentsUtils
+{
+    public static MeshesInfo GenerateMesh(IDCLEntity entity, Mesh mesh, GameObject gameObject,bool visible, bool withCollisions, bool isPointerBlocker)
```

Comment:
```suggestion
    public static MeshesInfo CreateMeshesInfo(IDCLEntity entity, Mesh mesh, GameObject gameObject,bool visible, bool withCollisions, bool isPointerBlocker)
```

Reply:
This generate a mesh and return the mesh info, create mesh info sound like only will get the info and not the mesh itself, lets wait to @AjimenezDCL to see which one is more appropiated

Reply:
Isn't the mesh generated by the `AssetPromiseKeeper_PrimitiveMesh`?

Reply:
Yes , my bad